materialize-databricks/driver.go

```diff
@@ -395,24 +398,41 @@ func (d *transactor) Store(it *m.StoreIterator) (_ m.StartCommitFunc, err error)
 		}
 		var fullPaths = pathsWithRoot(b.rootStagingPath, toCopy)
 
-		var query string
+		var queries []string
 		// In case of delta updates or if there are no existing keys being stored
 		// we directly copy from staged files into the target table. Note that this is retriable
 		// given that COPY INTO is idempotent by default: files that have already been loaded into a table will
 		// not be loaded again
 		// see https://docs.databricks.com/en/sql/language-manual/delta-copy-into.html
 		if b.target.DeltaUpdates || !b.needsMerge {
-			if query, err = RenderTableWithFiles(b.target, toCopy, b.rootStagingPath, tplCopyIntoDirect); err != nil {
-				return nil, fmt.Errorf("copyIntoDirect template: %w", err)
+			for i := 0; i < len(toCopy); i += queryBatchSize {
```

Comment:
Would be a nice place to use [slices.Chunk](https://pkg.go.dev/slices#Chunk), but that's not available until go1.23 which we haven't upgrade to yet, but maybe something to refactor when do we.

Reply:
That's nice, I did look into `slices` but didn't see Chunk. I'll add a comment so we do that later